src/main/java/net/eriagacha/RegisterCommands.java

```diff
@@ -45,8 +54,8 @@ public static void registerCommands() {
   }
 
 
-  public static void gachaRun(CommandContext<ServerCommandSource> ctx) throws CommandSyntaxException {
-    GachaController.giveItem(ctx);
+  public static void gachaRoll(CommandContext<ServerCommandSource> ctx, ItemStack moneyCondition) throws CommandSyntaxException {
+    GachaController.giveGachaReward(ctx, moneyCondition);
```

Comment:
I would rename this method to giveReward

Reply:
Done